qgispluginci/release.py

```diff
@@ -193,6 +194,18 @@ def create_archive(
                         continue
                     tt.add(m.name)
 
+    # add LICENSE if not already in plugin path but available in its parent
+    parent_folder_license_copied = False
+    if not Path(f"{parameters.plugin_path}/LICENSE").is_file():
+        parent_license = Path(f"{parameters.plugin_path}/../LICENSE")
+        if parent_license.is_file():
+            shutil.copy(
+                str(parent_license.resolve()), f"{parameters.plugin_path}/LICENSE"
```

Comment:
```suggestion
                parent_license.resolve(), f"{parameters.plugin_path}/LICENSE"
```

Minor, but `copy` accepts Pathlib object
https://docs.python.org/3/library/shutil.html#shutil.copy

Reply:
+1. And do you need to resolve it?